src/Bicep.Core/TypeSystem/Providers/Az/AzResourceTypeFactory.cs

```diff
@@ -146,14 +146,14 @@ private TypeSymbol ToTypeSymbol(Azure.Bicep.Types.Concrete.TypeBase typeBase, bo
                             isResourceBodyTopLevelPropertyType: isResourceBodyType));
 
                         return new ObjectType(objectType.Name,
-                            GetValidationFlags(isResourceBodyType, isResourceBodyTopLevelPropertyType),
+                            GetValidationFlags(isResourceBodyType, isResourceBodyTopLevelPropertyType, isSensitive: false),
```

Comment:
Objects may have been flagged as sensitive in RP swagger, too: https://github.com/Azure/bicep-types/blob/c1dc1b9a4bedd9b5ea25ad999d8cc3d810b2c112/src/Bicep.Types/Concrete/ObjectType.cs#L21